packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
I guess the `TemplatesSelectModal` is a part of the `templates` module, so let's move it.

Reply:
Moved in [7190298](https://github.com/mudita/mudita-center/pull/823/commits/719029829faf01967692767648a2d9a9ed576f54)